modules/builds-configmaps-use-case-consuming-in-volumes.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-consuming-in-volumes_{context}"]
+= Consuming ConfigMaps in volumes
+
+A ConfigMap can also be consumed in volumes. Returning again to the following example ConfigMap:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: special-config
+  namespace: default
+data:
+  special.how: very
+  special.type: charm
+----
+
+.Procedure
+
+You have a couple different options for consuming this ConfigMap in a volume.
+
+* The most basic way to consume a ConfigMap in a volume is to populate the volume with files where the key is the file name and the content of the file is the value of the key:
++
+[source,yaml]
+----
+apiVersion: v1
```

Comment:
I think it would be useful to include some call outs in this code block. This will help the reader to determine what is relevant to the mechanism being described. It might just be me, but when I look at fairly large YAML files, I find a bit of guidance useful in context to the topic being discussed.